dropwizard-client/src/test/java/io/dropwizard/client/DropwizardApacheConnectorTest.java

```diff
@@ -35,27 +37,31 @@
 import static org.hamcrest.CoreMatchers.any;
 
 public class DropwizardApacheConnectorTest {
-    private static final int SLEEP_TIME_IN_MILLIS = 500;
-    private static final int DEFAULT_CONNECT_TIMEOUT_IN_MILLIS = 200;
-    private static final int ERROR_MARGIN_IN_MILLIS = 300;
-    private static final int INCREASE_IN_MILLIS = 100;
-    private static final String NON_ROUTABLE_IP_ADDRESS = "10.255.255.1"; 
-    public static final URI URI_THAT_TIMESOUT_ON_CONNECTION_ATTEMPT = URI.create("http://" + NON_ROUTABLE_IP_ADDRESS);
+
+    private static final int SLEEP_TIME = 500;
```

Comment:
I think it was more clear when these had `_IN_MILLIS` in their names so that people would know what the value represented.